cfg/supervision-128k.cfg

```diff
@@ -33,5 +33,5 @@ SEGMENTS {
     ZEROPAGE: load = RAM,      type = bss, define   = yes;
     DATA:     load = RAM,      type = bss, define   = yes, offset = $0200;
     BSS:      load = RAM,      type = bss, define   = yes;
```

Comment:
* As far as I understand this makes the `BSS` segment start right after the `ZEROPAGE` segment. In case the `ZEROPAGE` segment shouldn't be exactly $100 bytes this means the `BSS` partly lives in the zeropage. Is that desired? Why?
* If the `ZEROPAGE` segment should be e.g. $20 bytes the `BSS` segment can't be larger than $1E0 bytes because the `DATA'  segment is fixed to offset $0200. Is that desired? Why?